src/EditorFeatures/Core/CommandHandlers/AbstractGoToCommandHandler.cs

```diff
@@ -1,133 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
this type was so substantially rewritten that i intentionally renamed the file to make this look like a delete/add.  That way all the new code is reviewed in isolatino, and not as a diff.